### PR TITLE
Added system test statement and additional tags

### PR DESCRIPTION
**Pull request by evelinec**

### CODE REVIEW

**Comment by evelinec** [comment]
Merged for small changes. We had already discussed about the tags in scrum this morning.

**Comment by evelinec** [comment]
@Kubik42 and @sishida Can you help review the changes in this PR? Sorry I close it too fast. I'll handle any feedback you may have.

File: README.adoc [hunk 1]
```diff
@@ -198,10 +201,9 @@ properties is correctly handled as an error.
 To force these test cases to run in a particular order, put them in a `testSuite()` method and label
 it with a `@Test` annotation so that it automatically runs when your test class run.
 
-Finally, notice that you're using a few helper methods throughout the tests.
-You can find complete Javadocs for these in the following `finish` directory for reference:
-`finish/src/test/java/it/io/openliberty/guides/inventory/EndpointTest.java`.
-
+Finally, a `system` test class in the `src/test/java/it/io/openliberty/guides/system/SystemEndpointTest.java` file,
+has been included for you to test the basic functionality of the `inventory` service.
```
Comment (Kubik42) [inline README.adoc]: Should be "... of the `system` service."

**Comment by sishida** [comment]
You don't need a comma after the word "file."
Also, I would say, "is included for you" instead of "has been included for you." 
:) Thanks!

File: README.adoc (hunk 1 above)
Comment (evelinec) [reply README.adoc]: Changed. Thanks.

**Comment by evelinec** [comment]
@sishida Fixed. Thanks for your review.

**Pull request merged**